src/style.css

```
@@ -0,0 +1,147 @@
+* {
+  margin: 0;
+}
+
+.photo {
+  display: flex;
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+  margin-bottom: 40px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+  width: 200px;
+  padding: 0 16px;
+}
+
+.card__title {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
+  box-sizing: border-box;
+  color: #060b35;
+}
+
+.card__code {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  align-self: start;
+  margin-top: 4px;
+}
+
+.rating {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-self: stretch;
+  margin-top: 16px;
+  align-items: center;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  margin-top: 24px;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  background: #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  height: 40px;
+  text-decoration: none;
+  color: #fff;
+  margin-bottom: 16px;
+  margin-top: 16px;
+}
+
+.card__button:hover {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  border: 1px solid #00acdc;
+  background: #fff;
+  color: #00acdc;
+  background-position: center;
+  width: 100%;
+  margin-bottom: 16px;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(/images/star-active.svg);
```

change only background-image, other styles haven't changed for active stars

```suggestion
  background-image: url(/images/star-active.svg);
```

---

also add all styles from stars task (for all cases, not only for 4 stars)